requirements.txt

```diff
@@ -1,5 +1,6 @@
 numpy >=1.16
-matplotlib ==3.1.1
+# matplotlib >=3.1.1,<=3.3.2 # For conda
+matplotlib ==3.3.2 # For the tests on travis
```

Comment:
Please use a patch or a sed command in the travis file, but keep the requirements.txt like it is in line 2.

Reply:
Or use a requirements-travis.txt file

Reply:
I thought requirements.txt was only for travis...

Reply:
There are people like me installing (especially for development) the required dependencies to an environment using `conda install --file requirements.txt` and `python setup.py install` :)

Reply:
Me too but then when you are developing you want the version as in travis, no? I don't mind to split it...